### DISCUSSION TITLE
Screen blank issues

### DISCUSSION THREAD

**Discussion by annabelhammond** [microsoft/vsts-extension-retrospectives#788] [General]
Has anyone else experienced this? As title suggests- my retrospectives board screen shows blank everytime I load it, I have tried everything from removing myself and readding, opening in private browser, removing cookies and clearing cache etc- any advise on what I can do???

**Comment by SegFault1963** [comment]
@annabelhammond I am the administrator for our on-prem Azure Devops Server instance, and one of my users reported this problem this week. It is somewhat alarming to see that it goes back to July (it is now October 21)

**Reply by annabelhammond** [reply]
If it helps the issue resolved itself - unfortunately I have no idea why its now resolved.

**Comment by beedrthomas** [comment]
Yep, common issues we've seen so far:
- Screen crashes and shows the gray screen with the white old mac unhappy face after a while.
- Screen opens blank, then after a while it fills.
- Error "board is taking longer than usual to load"

And these we have experienced for more than a year, and havent been addressed so far. Im fairly sure others have reported them as well, but .... yeah.
